src/licenseUtils.js

```diff
@@ -18,23 +18,34 @@ const getLicenseContents = dependencyPath => {
   return licensePath && wrap(readFileSync(licensePath).toString(), licenseWrap);
 };
 
-const getLicenseName = package => {
-    if (package.licenses) {
-        const licenseName = package.licenses.map(license => license.type).join(" OR ");
-        return package.licenses.length > 1 ? `(${licenseName})` : licenseName;
-    }
-
-    return package.license;
-}
+const getLicenseName = pkg => {
+  // Valid license formats are an SPDX string expression
+  // https://www.npmjs.com/package/spdx
+  if (typeof pkg.license === "string") {
+    return pkg.license;
+  }
+  // Check for deprecated licence formats
+  // https://docs.npmjs.com/files/package.json#license
+  if (typeof pkg.license === "object") {
+    return pkg.license.type || "";
+  }
+  if (pkg.licenses && Array.isArray(pkg.licenses)) {
+    const licenseName = pkg.licenses.map(license => license.type).join(" OR ");
+    // Return the list of licenses as a composite license expression
+    return pkg.licenses.length > 1 ? `(${licenseName})` : licenseName;
+  }
+  // Fall-back to an empty string and let the validation handle the error.
+  return "";
+};
 
 const getLicenseInformationForDependency = dependencyPath => {
-  const package = require(`${dependencyPath}/package.json`);
+  const pkg = require(`${dependencyPath}/package.json`);
```

Comment:
This is because jest/babel complains that `package` is a reserved word

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/1110551/52373323-b692ad80-2a5a-11e9-93ab-9355502739c9.png">